app/models/regional_holiday.rb

```diff
@@ -2,14 +2,10 @@
 
 class RegionalHoliday < ApplicationRecord
   has_and_belongs_to_many :cities
-  has_and_belongs_to_many :offices
```

Comment:
the migration, to drop the table is missing

Reply:
This migration was already merged in this PR: https://github.com/Codeminer42/Punchclock/pull/398/files#diff-7512307fbce008e0793ceff6f860d8dd6649aa2e7687be9d82e27261868bbe48

The removed code was meant to keep legacy registers working, but I reckon it's not needed anymore.